src/digidoc-tool.cpp

```diff
@@ -316,6 +316,11 @@ static void printUsage(const char *executable)
     << "      --mime=        - can be after --file parameter. Default value is application/octet-stream" << endl
     << "      --dontsign     - Don't sign the newly created container." << endl
     << "      for additional options look sign command" << endl << endl
+    << "  Command createPatch:" << endl
```

Comment:
Could perhaps use a clearer name, it's probably only programmers who associate patch with actually patching something (no idea what normal people think though), but I would probably prefer something like "signAllFilesInFolder" but then again that could mislead to believe that all the files end up inside the same container...